### PR TITLE
Add support for admin to customize login layout

### PR DESCRIPTION
**Pull request by Sae126V**
Resolves https://github.com/indigo-iam/iam/issues/298.

TODO:

Update https://indigo-iam.github.io/ with relevant information.

### CODE REVIEW

File: iam-login-service/src/main/webapp/WEB-INF/views/iam/login-form.jsp [hunk 1]
```diff
@@ -19,8 +19,22 @@
...
+</c:if>
+
+<form id="login-form" class="${containerSpacer}" action="/login" method="post">
+  <div class="signin-preamble text-muted">
+    <c:choose>
+      <c:when test="${loginPageConfiguration.defaultLoginPageLayout}">
+        Sign in with your ${iamOrganisationName} credentials
+      </c:when>
+      <c:otherwise>
+        Or sign in with your ${iamOrganisationName} credentials
```
Comment (giacomini) [inline iam-login-service/src/main/webapp/WEB-INF/views/iam/login-form.jsp:33]: "Sign in" again?

File: iam-login-service/src/main/webapp/WEB-INF/views/iam/login.jsp [hunk 2]
```diff
@@ -72,62 +72,129 @@
             <h3>Welcome to <strong>${iamOrganisationName}</strong>
       </h3>
         </div>
+
+        <c:if
+            test="${
+                (
+                    loginPageConfiguration.localAuthenticationVisible
+                    or param.sll != null
```
Comment (giacomini) [inline iam-login-service/src/main/webapp/WEB-INF/views/iam/login.jsp:80]: Can `or` be `||` (for uniformity)?

File: iam-login-service/src/main/webapp/WEB-INF/views/iam/login.jsp [hunk 3]
```diff
@@ -72,62 +72,129 @@
...
+            }"
+        >
+            <jsp:include page="login-form.jsp" />
+        </c:if>
         
-        <c:if test="${loginPageConfiguration.localAuthenticationVisible or param.sll != null}">
-          <jsp:include page="login-form.jsp" />
+        <c:if
+            test="${
+                !loginPageConfiguration.defaultLoginPageLayout
```
Comment (giacomini) [inline iam-login-service/src/main/webapp/WEB-INF/views/iam/login.jsp:89]: Isn't this equivalent to `!loginPageConfiguration.defaultLoginPageLayout || !loginPageConfiguration.localAuthenticationVisible && param.ssl == null`?

File: iam-login-service/src/main/webapp/WEB-INF/views/iam/login.jsp [hunk 4]
```diff
@@ -72,62 +72,129 @@
...
             <c:if
-        test="${loginPageConfiguration.oidcEnabled or loginPageConfiguration.samlEnabled or IAM_X509_CAN_LOGIN}">
-              <div class="ext-login-preamble text-muted">
+                test="${loginPageConfiguration.oidcEnabled or loginPageConfiguration.samlEnabled or IAM_X509_CAN_LOGIN}"
+            >
+                <div class="ext-login-preamble text-muted">
+                    <c:choose>
+                        <c:when
+                            test="${
+                                (
```
Comment (giacomini) [inline iam-login-service/src/main/webapp/WEB-INF/views/iam/login.jsp:107]: Isn't this equivalent to `loginPageConfiguration.defaultLoginPageLayout && (loginPageConfiguration.localAuthenticationVisible || param.sll != null)`

File: iam-login-service/src/main/webapp/WEB-INF/views/iam/login-form.jsp (hunk 1 above)
Comment (Sae126V) [reply iam-login-service/src/main/webapp/WEB-INF/views/iam/login-form.jsp:33]: In our scenario, Assuming that user want external authn block to be displayed first, Now for our local credentials section, we have to change the text to `or sign in`. Does that make sense, ?

File: iam-login-service/src/main/webapp/WEB-INF/views/iam/login.jsp (hunk 2 above)
Comment (Sae126V) [reply iam-login-service/src/main/webapp/WEB-INF/views/iam/login.jsp]: Sure..Updated the code to be consistent,

File: iam-login-service/src/main/webapp/WEB-INF/views/iam/login.jsp (hunk 3 above)
Comment (Sae126V) [reply iam-login-service/src/main/webapp/WEB-INF/views/iam/login.jsp:89]: Simplified the logic. Done

File: iam-login-service/src/main/webapp/WEB-INF/views/iam/login.jsp (hunk 4 above)
Comment (Sae126V) [reply iam-login-service/src/main/webapp/WEB-INF/views/iam/login.jsp]: Simplified the logic. Done

**Pull request merged**